### PR TITLE
Added example HEALPIX map (simulation for aLIGO)

### PR DESCRIPTION
**Pull request by astrofrog**
@lpsinger - does the description seem accurate to you? (since I believe you are involved in this project!)

This will add the file to http://data.astropy.org and make it accessible from example scripts.

### CODE REVIEW

**Comment by astrofrog** [comment]
I'm going to go ahead and merge this since I need it for writing the reproject package docs, but let me know if anything in the description should be updated!

File: allsky/README.md [hunk 1]
```diff
@@ -2,3 +2,10 @@ All-sky
 =======
 
 * ``allsky_rosat.fits``: [ROSAT](http://heasarc.gsfc.nasa.gov/docs/rosat/rass.html) Soft X-ray Diffuse Background in the 3/4 keV band, in an Aitoff projection
+
+* ``ligo_simulated.fits.gz``:
+  [All-sky posterior map](http://www.ligo.org/scientists/first2years/) in
+  HEALPIX format for a simulated event observed by the Advanced Laser
+  Interferometer Gravitational-wave Observatory (aLIGO). The event ID is
+  890025 and the map was produced by Bayestar. The maps are described in more
```
Comment (lpsinger) [inline allsky/README.md:10]: It's actually spelled in all caps ("BAYESTAR").

File: allsky/README.md [hunk 2]
```diff
@@ -2,3 +2,10 @@ All-sky
...
+  detail in [Berry et al. (2014)](http://arxiv.org/abs/1411.6934).
```
Comment (lpsinger) [inline allsky/README.md:11]: This particular data file is from `[Singer et al. (2014)](http://arxiv.org/abs/1404.5623)`.

**Comment by astrofrog** [comment]
@lpsinger - thanks, fixed in 76a9445dd5a435c13665b3b03c7449b82da0a9e0

**Pull request merged**